lib/widgets/audio_bottom_sheet/audio_minimized_info.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:flutter/material.dart';
```

Comment:
imo, kurangin wrap widget di suatu method. Jadi mungkin alternative lain ini bisa dipisah ke folder audio_minimized_info terus di dalemnya ada file ini dan yang buildnya dari method bisa dibuat file tersendiri dengan class widget tersendiri cmiiw. Tapi kalo ga sempet comment jadi tech debt aja source: https://docs.flutter.dev/perf/best-practices

Reply:
okee ndras thank you infonya.. refactoring

Reply:
<img width="662" alt="image" src="https://github.com/Yaumi-Digital-School/quranplus-flutter/assets/36220334/4e20c405-f8ac-406d-a0dd-2d3ab38ed8ce">

Updating ndras, thank you!

Reply:
terkait ini gamau dipisah di beberapa file aja yum? takutnya nanti semakin melebar ukuran line di file nya cmiiw. Jadi mungkin bisa di strict one file one class kedepannya(?), buat ngehindari membengkak aja sih dan fokus gitu imho

Reply:
Oke boleh ndras, tp kyknya perlu kita bikin standar jg, coba kita diskusiin yak

Apakah hrs setiap class ? 
klo menurut gue pribadi, gk usah spesifik tiap class.. krn bisa jadi kita butuh bikin private class yg sebenernya dia itu helper class aja (buat tampilan, gaada logic yg berkaitan sm state/data) yg spesifik bwt halaman tsb. Nah utk yg kyk gt, kyk nya gak butuh deh (?) misal bwt nampilin icon aja kyk yg gue buat inii dan spesifik bwt bantu widget ini aja.

Tapi untuk yang didalamnya ada logic, gue setuju sih bwt dipisah jd class baru spy merampingkan isinya aja.. Misal disini dynamic icon itu ada logic tuh, itu gue pisah.. tp si dynamic icon ini didalamnya butuh nampilin icon dengan size tertentu, jd gue butuh helper class yg sifatnya private aja yg scope nya gk besar.

Nah klo ada pendapat lain sabiii yaaa